vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/ProjectFormLayout.java

```diff
@@ -74,6 +77,23 @@ public ProjectFormLayout() {
         .bind((ProjectInformation::getProjectObjective),
             ProjectInformation::setProjectObjective);
 
+    fundingField = new FundingField("Funding Information");
+    binder.forField(fundingField)
+        .withValidator(field -> {
+              if (field == null) {
+                return true;
+              }
+              if (!field.getLabel().isBlank()) {
+                return !field.getReferenceId().isBlank();
+              } else if (!field.getReferenceId().isBlank()) {
+                return !field.getLabel().isBlank();
+              }
+              return true;
```

Comment:
Should be split up into different validators to allow for different error messages depending on the cause. Right now it is not possible to say something like `please provide the funding reference id` when only funding label is filled out.